views/partials/header.pug

```diff
@@ -0,0 +1 @@
+h1 Kim's Convenience
```

Comment:
Could have pulled more out into this partial. Your head and link tags are also repeated.